src/v2/Components/CountrySelect.tsx

```diff
@@ -6,6 +6,7 @@ export interface CountrySelectProps extends PositionProps, SpaceProps {
   selected?: string
   disabled?: boolean
   euShippingOnly?: boolean
+  error?: string
```

Comment:
`props` are passed down to `LargeSelect` which has `error` prop so just had to add it to prop types